tests/json-fixtures/test_transactions.py

```diff
@@ -102,9 +102,9 @@ def fixture_transaction_class(fixture_data):
         return ByzantiumTransaction
     elif fork_name == ForkName.Constantinople:
         return ConstantinopleTransaction
-    elif fork_name == "Petersburg":
+    elif fork_name == "ConstantinopleFix":
```

Comment:
created https://github.com/ethereum/eth-typing/pull/15 in order to be consistent with `ForkName.Name`.

Reply:
The PR is merged. I've asked piper to cut a release in `eth-typing` so that we can use it here.

Reply:
@veox Here's the release: https://pypi.org/project/eth-typing/2.2.0/